phpunit-plugin-bootstrap.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+require_once( __DIR__ . '/lib/antecedent/patchwork/Patchwork.php' );
```

Comment:
Heres a pull request for the project specific phpunit bootstrap: https://github.com/xwp/wp-dev-lib/pull/114